src/components/CreateVmWizard/steps/SummaryReview.js

```diff
@@ -80,12 +81,14 @@ const ReviewNetworking = ({ id, vnicProfiles, network }) => {
     return map
   }, {})
 
+  const nicsList = sortNicsDisks([...network]) // Sort the template based ones first, then by name
```

Comment:
This could just as easily be done in the `SummaryReview` component and passed in as a prop.  You wouldn't need to change the `ReviewNetworking` or `ReviewStorage` components in that case.

Reply:
Yeah, maybe, I'll try. And what about using the _slice()_ for the array of NICs/Disks instead of spread operator? WDYT? Thanks!

Reply:
IMHO spread operator looks better then other variants. Unless we plan to have huge list I would stick to it.

Reply:
Radek makes a good point.  Stick with the spread.